.github/workflows/conformance-suites.yml

```diff
@@ -2,18 +2,29 @@ name: Run Xbrl Conformance Suites
 on: pull_request_target
 
 jobs:
+  find-tests:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.build-test-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          ref: ${{ github.event.pull_request.head.sha }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - id: build-test-matrix
+        run: |
+          printf '::set-output name=matrix::'
+          find tests/integration_tests -name 'test_*.py' -not -name test_efm_conformance_suite.py | jq -Rsc 'split("\n") | map(select(length > 0))'
+
   run-conformance-suite:
+    needs: find-tests
     permissions: read-all
     runs-on: ubuntu-latest
     continue-on-error: true
     strategy:
       matrix:
-        test-path:
```

Comment:
This doesn't really seem like an objective improvement, would suggest not doing this, but will leave it up to the team. My concerns are when we want to change, you now have to know `find`, and `jq`. I will leave up to the team though

Reply:
It is more complicated, but the value it adds (running your PR's new tests before merging) is worth it in my opinion.

Reply:
I'm personally fine with both `find` and `jq`. They're both pretty standard utils